src/Umbraco.Cms.Integrations.Automation.Zapier/Components/NewContentPublishedComponent.cs

```diff
@@ -2,18 +2,66 @@
 using System.Collections.Generic;
 using System.Net.Http;
 using System.Threading.Tasks;
+
```

Comment:
I think this file could be amended a bit, as it's a bit difficult to follow given it's pretty much the whole class that changes, and the name of the class itself varies between the two versions.

Would suggest having two classes and files - and just fully include/exclude the whole file based on the compiler directive, like [this one](https://github.com/umbraco/UI-Examples/blob/master/src/Our.Umbraco.UiExamples/Migrations/UmbracoAppStartingHandler.cs).